.github/workflows/test.yaml

```diff
@@ -0,0 +1,24 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  Test:
+    runs-on: ubuntu-20.04
```

Comment:
I think conformance test also needs docker daemon to be running. Is it available on `ubuntu-20.04` by default.